modules/installation-gcp-user-infra-wait-for-bootstrap.adoc

```diff
@@ -14,11 +14,7 @@ installation program.
 
 .Prerequisites
 
-* Configure a GCP account.
-* Generate the Ignition config files for your cluster.
-* Create and configure a VPC and associated subnets in GCP.
-* Create and configure networking and load balancers in GCP.
-* Create control plane and compute roles.
+* Ensure you defined the variables in the _Exporting common variables_ section.
```

Comment:
FYI some variables referenced in this section, `ZONE_0`, is defined in the section "Creating load balancers in GCP", rather than in "Exporting common variables" section. So?